packages/synapse-react-client/src/lib/containers/oauth/OAuthManagement.tsx

```diff
@@ -180,17 +180,16 @@ export const OAuthManagement: React.FunctionComponent = () => {
         <Modal.Footer></Modal.Footer>
       </Modal>
 
-      <WarningModal
-        show={isShowingSecretWarning}
-        className="bootstrap-4-backport"
```

Comment:
This is worth a spot-check -- the modal may contain Bootstrap 4 components that still require this class.

Reply:
Good call -- since`warningHeader` and `warningBody` are both strings, I don't think they require this class -- 

main | feature
:---:|:---:
<img width="1353" alt="main_OAuthManagement" src="https://user-images.githubusercontent.com/26949006/233208523-57bb9d5f-420e-40ef-9182-498f5a02aace.png"> |  <img width="1351" alt="feature_OAuthManagement" src="https://user-images.githubusercontent.com/26949006/233208521-fb391530-adab-48e5-ada1-c1e4257b10a5.png">